src/System.Reflection.Metadata/tests/Resources/Interop/README.md

```diff
@@ -0,0 +1,15 @@
+# System.Reflection.Metadata Tests
```

Comment:
Why do we need the same README five times?  Couldn't we just put it up a level, in Resources, instead of duplicating it in each of the subdirectories under Resources?